packages/api/helper/mailer.js

```diff
@@ -36,13 +36,25 @@ const email = new emailTemplate({
   preview: false,
 });
 
+function validateEmail(email) {
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(email).toLowerCase());
+}
+const recEmail = (nomEmail) => {
+  if(process.env.SEND_TO_TEST && validateEmail(process.env.RECIPIENT_EMAIL)){
```

Comment:
I don't think we need a separate `SEND_TO_TEST` variable. Instead we can just check if `RECIPIENT_EMAIL` is not null and has a length greater than zero.

Reply:
Removed SEND_TO_TEST from if statement.